core/appeal/service.go

```diff
@@ -972,8 +888,7 @@ func (s *Service) handleAppealRequirements(ctx context.Context, a *domain.Appeal
 	return nil
 }
 
-func (s *Service) CreateAccess(ctx context.Context, a *domain.Appeal, opts ...CreateAppealOption) error {
-	// TODO: rename to GrantAccess
+func (s *Service) GrantAccessToProvider(ctx context.Context, a *domain.Appeal) error {
```

Comment:
why do we remove the options from this func parameter?

Reply:
previously removing because the `opts` arguments never passed anywhere on this [commit](https://github.com/odpf/guardian/commit/015bec7020d4613e61f26d7e0e0879d683dbd681), seems unused and make the mocking the function become complex,

should we add this back?

Reply:
It was added [here](https://github.com/odpf/guardian/pull/222/files#diff-4ceaf11357ea8ac162487cfbb8f9f0a0a7446fa7b61bc799aac1a3d3ca7adc58R264) and got accidentally removed [here](https://github.com/odpf/guardian/pull/249/files#diff-4ceaf11357ea8ac162487cfbb8f9f0a0a7446fa7b61bc799aac1a3d3ca7adc58R282) I think. @bsushmith could you help here?

Reply:
nice catch mas @rahmathd, added the opts back